extensions/rocksdb-repos/ProvenanceRepository.h

```diff
@@ -99,9 +90,26 @@ class ProvenanceRepository : public core::Repository, public std::enable_shared_
     options.create_if_missing = true;
     options.use_direct_io_for_flush_and_compaction = true;
     options.use_direct_reads = true;
-    rocksdb::Status status = rocksdb::DB::Open(options, directory_, &db_);
+    int64_t max_buffer_size = 16 << 20;
```

Comment:
What's the origin of this constant: `16 << 20`? Code comment would be appreciated.

Reply:
Added comment.